gnc/navigator_perception/exFAST_SparseStereo/GPL.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Well I used a feature detector that was open sourced under this license and asked for it to be included in derivatives. I guess we could just have a link to it or something like that.